CMakeLists.txt

```diff
@@ -461,6 +461,16 @@ add_subdirectory(config)
 add_subdirectory(test/functional/fixtures) # compile pty/shell test programs
 add_subdirectory(runtime)
 
+
+# add 'desktop' target to install neovim.desktop
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Should go in `runtime/CMakeLists.txt` and installed as part of the `runtime` target.

instead of checking system name, how about checking if `xdg-desktop-menu` is available?

```
find_program(XDG_MENU_PRG xdg-desktop-menu)
find_program(XDG_ICON_PRG xdg-icon-resource)

if(XDG_MENU_PRG)
...
endif()

if(XDG_ICON_PRG)
...
endif()

```